Mathlib/AlgebraicGeometry/EllipticCurve/Affine.lean

```diff
@@ -639,7 +639,7 @@ inductive Point
 #align weierstrass_curve.point WeierstrassCurve.Affine.Point
 
 /-- For an algebraic extension `S` of `R`, the type of nonsingular `S`-rational points on `W`. -/
-scoped notation3 W "⟮" S "⟯" => Affine.Point <| baseChange W S
+scoped[WeierstrassCurve] notation3:max W "⟮" S "⟯" => Affine.Point <| baseChange W S
```

Comment:
Seems like there's no need for `[WeierstrassCurve]` since it's already in the namespace:
```suggestion
scoped notation3:max W "⟮" S "⟯" => Affine.Point <| baseChange W S
```